Neos.Flow/Classes/Http/ContentStream.php

```diff
@@ -0,0 +1,314 @@
+<?php
+namespace Neos\Flow\Http;
+
+use Psr\Http\Message\StreamInterface;
+
+/**
+ * Implementation of a PSR-7 HTTP stream
+ */
+class ContentStream implements StreamInterface
+{
+    /**
+     * @var resource
+     */
+    protected $resource;
+
+    /**
+     * @var string|resource
+     */
+    protected $stream;
+
+    /**
+     * @param string|resource $stream
+     * @param string $mode Mode with which to open stream
+     * @throws \InvalidArgumentException
+     */
+    public function __construct($stream, $mode = 'r')
+    {
+        $this->replace($stream, $mode);
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
why do we have inheritdoc here, but not on the other methods? I'd pefer to have actual doc comments here.